mjcf_to_sdformat/tests/resources/tennis_ball.xml

```diff
@@ -0,0 +1,19 @@
+<mujoco model="dog">
+  <asset>
+    <texture name="tennis_ball" file="tennis_ball.png" gridsize="3 4" gridlayout=".U..LFRB.D.."/>
```

Comment:
I don't think an empty file will work well when generating the sha1 hash, so I 
suggest adding an actual texture.

Reply:
I believe the file is in the `resources` folder